adal/src/main/java/com/microsoft/aad/adal/Discovery.java

```diff
@@ -54,21 +55,21 @@
 
     private static final String API_VERSION_KEY = "api-version";
 
-    private static final String API_VERSION_VALUE = "1.0";
+    private static final String API_VERSION_VALUE = "1.1";
 
     private static final String AUTHORIZATION_ENDPOINT_KEY = "authorization_endpoint";
 
     private static final String INSTANCE_DISCOVERY_SUFFIX = "common/discovery/instance";
 
     private static final String AUTHORIZATION_COMMON_ENDPOINT = "/common/oauth2/authorize";
 
-    private static final String TENANT_DISCOVERY_ENDPOINT = "tenant_discovery_endpoint";
+    private volatile static ReentrantLock sLock;
```

Comment:
Can you add a comment explaining exactly what this lock is for, or better yet a more descriptive variable name?

Reply:
Sure. This one is already merged, I'll update in another PR.